src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1380,7 +1378,8 @@ private FileSystemInfo GetFileSystemItem(string path, ref bool isContainer, bool
 
             // FileInfo.Exists is always false for a directory path, so we check the attribute for existence.
             var attributes = result.Attributes;
-            if ((int)attributes == -1) { /* Path doesn't exist. */ return null; }
+            if ((int)attributes == -1)
+            { /* Path doesn't exist. */ return null; }
```

Comment:
Please revert such unrelated style changes here and below too.

Reply:
I'll do so. Do you know of a way I can tell Visual Studio's ctrl+k,ctrl+d about these style preferences?

Reply:
I don't know about Visual Studio but we try to put all needed options to EditorConfig file.

Reply:
You/we can perhaps add some config files to help the autoformatters do better work, but there are just some older files in the repo that haven't had a formatting pass in quite some time, so it's not always avoidable.

Sometimes it may make sense to just do a strict format pass over the files at least once in a PR before doing a feature/bugfix change PR if it hasn't been updated in a while, but we don't necessarily expect every small change to force a complete file reformat as well, so whether you want to do that extra bit is really up to you.

Reply:
Latest commit has all of these style changes reverted.